modynclient/client/client.py

```diff
@@ -169,3 +170,24 @@ def poll_pipeline_status(self) -> None:
             logger.info(f"Pipeline <{self.pipeline_id}> not found.")
         else:
             logger.error(f"unknown pipeline status {json.dumps(res, sort_keys=True, indent=2)}")
+
+
+def run_multiple_pipelines(
```

Comment:
Not sure this should go into the implementation file of the modyn client (since it uses client) Should this be part of the CLI instead maybe? What was the reasoning for putting it here?

Reply:
Maybe this even is should an experiments util function?

Reply:
It's not really an entrypoint. I like the experiment util idea!